conman/src/scheme.cpp

```diff
@@ -1759,6 +1802,39 @@ bool Scheme::disableBlocks(
   return success;
 }
 
+bool Scheme::disableBlocksTopo(
```

Comment:
In this PR, can you just replace `disableBlocks` with the contents of `disableBlocksTopo`?